addons/website_slides/tests/test_security.py

```diff
@@ -152,6 +152,52 @@ def test_access_slide_preview(self):
         self.slide.with_user(self.user_portal).read(['name'])
         self.slide.with_user(self.user_public).read(['name'])
 
+    @mute_logger('odoo.models', 'odoo.addons.base.models.ir_rule')
+    def test_access_channel_visibility_public(self):
```

Comment:
can we add more tests with website_published set to False ? (you can just check the read access)